### PR TITLE
chre: add fix for esp32s2_saola platform

### PR DESCRIPTION
**Pull request by yperess**
Add missing source needed for esp32s2_saola cross compiler

Signed-off-by: Yuval Peress <peress@google.com>

### CODE REVIEW

File: west.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ manifest:
       revision: 53d3415c14d60f8f4bfca54bfbc5d5a667d7e724
       path: modules/lib/canopennode
     - name: chre
-      revision: ef76d3456db07e4959df555047d6962279528c8d
+      revision: pull/6
```
Comment (stephanosio) [inline west.yml:32]:
```suggestion
      revision: pull/6/head
```

**Comment by yperess** [comment]
Squashed the commits (which should resolve the compliance checks)

**Comment by stephanosio** [comment]
@yperess https://github.com/zephyrproject-rtos/chre/pull/6 has been merged. Please update the ref.

**Comment by yperess** [comment]
Done

> @yperess [zephyrproject-rtos/chre#6](https://github.com/zephyrproject-rtos/chre/pull/6) has been merged. Please update the ref.

Done

**Review by fabiobaltieri — APPROVED** [review]

**Review by stephanosio — APPROVED** [review]

**Pull request merged**